src/dotnet/APIView/APIView/CodeLine.cs

```diff
@@ -6,10 +6,13 @@ namespace ApiView
     {
         public string DisplayString { get; }
         public string ElementId { get; }
-        public CodeLine(string html, string id)
+
+        public bool isLineAllDocumentation { get; }
```

Comment:
Formatting and naming
```suggestion
        public bool IsDocumentationLine { get; }

```